awx/sso/fields.py

```diff
@@ -233,12 +234,34 @@ def _default_from_required_settings(self):
 
 class LDAPServerURIField(fields.URLField):
 
+    tld_re = (
+        r'\.'                                              # dot
+        r'(?!-)'                                           # can't start with a dash
+        r'(?:[a-z' + URLValidator.ul + r'0-9' + '-]{2,63}' # domain label
```

Comment:
Is this what changed from the original Django class? It might be good to point that out in a comment right here.

Reply:
I made a comment to indicate which line was changed from django URLValidator